contrib/depends/packages/bdb.mk

```diff
@@ -1,6 +1,6 @@
 package=bdb
```

Comment:
Uh... Why is BDB a dependency?

Reply:
I have a branch where the bdb package is removed, but since there are references to it all over the codebase, I think a general removal of it would make more sense.

Reply:
It's never selected. I'd be fine with it being removed.